Sprint-1/errors/4.js

```diff
@@ -1,2 +1,2 @@
-const 12HourClockTime = "20:53";
-const 24hourClockTime = "08:53";
\ No newline at end of file
+const hourClockTime12 = "20:53";
+const hourClockTime24 = "08:53";
```

Comment:
Assessment of problem is correct, but there seem further issues with the original variable declarations. I suggest taking a closer at the values of the variables in relation to the variable names to find the issue.

Reply:
Right,  thank you,  the variable names and values were mismatched. 
It should be :
`const hourClockTime24 = "20:53";
const hourClockTime12 = "08:53";`